src/Symfony/Bundle/FrameworkBundle/Translation/TranslationLoader.php

```diff
@@ -11,56 +11,9 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Translation;
 
-use Symfony\Component\Finder\Finder;
-use Symfony\Component\Translation\MessageCatalogue;
-use Symfony\Component\Translation\Loader\LoaderInterface;
-
 /**
- * TranslationLoader loads translation messages from translation files.
- *
- * @author Michel Salib <michelsalib@hotmail.com>
+ * @deprecated Class moved to Symfony\Component\Translation\Loader\TranslationLoader
  */
-class TranslationLoader
+class TranslationLoader extends \Symfony\Component\Translation\Loader\TranslationLoader
```

Comment:
And internal usages of this class should be changed to use the new class instead (which could lead to some composer dep changes in frameworkbundle probably)